contracts/Controller.sol

```diff
@@ -25,7 +31,7 @@ contract Controller is Ownable {
     address internal addressBook;
 
     /// @dev mapping between owner address and account structure
```

Comment:
can you also add an array or some way to know on chain all the acocunt owners? (need it for invariant testing that all the collateral sums on all vaults <= total collateralAsset in margin pool)

Reply:
I think we don't need for a functionality we shouldn't add it! thoughts @antoncoding ?

Reply:
@aparnakr If that's for fuzz testing, can't we only have that in the echidna contract? 
I think we shouldn't add them. Arrays on chain are pretty dangerous.

Reply:
hmm we would basically need to add to that array every time an account was created for echidna. What is the dangerous part of having it on chain?

Reply:
we can have a different contract we are running echidna on, but every additional change to the contract which the echidna contract is running on is a little more out of date compared to the one we are deploying, so we ideally don't want to have too much different data